datasets/il/mod_crypto/crawler.py

```diff
@@ -1,125 +1,135 @@
+        wallet.set("publicKey", account_id)
+        platform = row.pop("platform")
+        if platform:
             wallet.set("mangingExchange", platform)
-            wallet.set("publicKey", account_id)
-            wallets.append(wallet)
-    if wallet_ids := row.pop("Wallet ID"):
-        for wallet_id in h.multi_split(wallet_ids, [";"]):
-            d = h.extract_cryptos(wallet_id)
-            wallet = context.make("CryptoWallet")
-            wallet.id = context.make_id(wallet_id)
-            wallet.set("publicKey", wallet_id)
-            wallets.append(wallet)
-            if wallet_id in d:
-                wallet.set("currency", d[wallet_id])
-
-    # Create a sanction for each Crypto account
-    aso_id = row.pop("Administrative Seizure Order")
-    source_url = row.pop("sourceUrl")
-    start_date = row.pop("Publication date")
-    end_date = row.pop("ASO valid until")
-    for wallet in wallets:
-        wallet.set("holder", person)
+        currency = row.pop("currency")
+        if currency:
+            wallet.set("currency", currency)
+        if person or entity:
+            wallet.set("holder", person or entity)
+        wallets.append(wallet)
 
+    # --- Sanction & Linking ---
+    aso_id = row.pop("order_id")
+    # TODO: startDate is to be extracted from the orders
+    # start_date = row.pop("Publication date")
+    for wallet in wallets:
         sanction = h.make_sanction(context, wallet, key=aso_id)
         sanction.set("authorityId", aso_id)
-        h.apply_date(sanction, "startDate", start_date)
-        h.apply_date(sanction, "endDate", end_date)
+        # h.apply_date(sanction, "startDate", start_date)
+        h.apply_date(sanction, "modifiedAt", row.pop("last_updated"))
+        h.apply_date(sanction, "endDate", row.pop("end_date"))
         if h.is_active(sanction):
             wallet.add("topics", "crime.terror")
-        sanction.add("sourceUrl", source_url)
+        sanction.add("sourceUrl", row.pop("order_url"))
+        sanction.add("sourceUrl", row.pop("forfeiture_order_url"))
+        sanction.add("sourceUrl", row.pop("annex_url"))
         context.emit(wallet)
         context.emit(sanction)
 
-    context.audit_data(
-        row,
-        ignore=[
-            # Just sequencing in the spreadsheet, not from the source.
-            "#",
-            # The wording is "the property of the organization, or used in financing
-            # terrorism", i.e. something in the doc belongs to the org, but not
-            # necessarily everything.
-            "Organization",
-        ],
-    )
+    context.audit_data(row)
 
 
 def crawl(context: Context):
     # Get a warning when a notice has been issued
     content_xpath = ".//main"
     doc = fetch_html(context, context.dataset.url, content_xpath, cache_days=1)
     container = doc.xpath(content_xpath)[0]
+    check_csv_diff(context, doc)
 
     # The key things to check are
     # - the table of releases - are there any new ones?
```

Comment:
Let's update this documentation here. IIUC, we can now `git diff` on the CSV to detect changes, and then update the Google Sheet manually, right?